### PR TITLE
Add ForemanLeapp plugin

### PR DESCRIPTION
**Pull request by stejskalleos**
Packaging PR: https://github.com/theforeman/foreman-packaging/pull/4986

### CODE REVIEW

File: spec/classes/plugin/leapp_spec.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::leapp' do
+  include_examples 'basic foreman plugin tests', 'leapp'
+  it { should contain_foreman__plugin('remote_execution') }
```
Comment (ares) [inline spec/classes/plugin/leapp_spec.rb:5]: this should say leapp

File: manifests/plugin/leapp.pp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Installs foreman_leapp plugin
+class foreman::plugin::remote_execution {
+  include ::foreman::plugin::remote_execution
```
Comment (ares) [inline manifests/plugin/leapp.pp:3]: leapp here and on the line above

File: manifests/plugin/leapp.pp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Installs foreman_leapp plugin
+class foreman::plugin::remote_execution {
```
Comment (ares) [inline manifests/plugin/leapp.pp]: class should have name leapp, not rex

**Review by ares — APPROVED** [review]
LGTM, packaging PR is merged https://github.com/theforeman/foreman-packaging/pull/4986

**Comment by stejskalleos** [comment]
Updated for https://github.com/theforeman/foreman-packaging/pull/5066 - Added Ansible dependency

**Pull request closed without merging**